backend/service/github/github.go

```diff
@@ -362,21 +362,32 @@ func (s *svc) CompareCommits(ctx context.Context, ref *RemoteRef, compareSHA str
 }
 
 type Commit struct {
-	Files   []*githubv3.CommitFile
-	Message string
-	Author  *githubv3.CommitAuthor
+	Files           []*githubv3.CommitFile
+	Message         string
+	Author          *githubv3.CommitAuthor
+	AuthorAvatarURL *string
+	AuthorID        int64
 }
 
 func (s *svc) GetCommit(ctx context.Context, ref *RemoteRef) (*Commit, error) {
 	commit, _, err := s.rest.Repositories.GetCommit(ctx, ref.RepoOwner, ref.RepoName, ref.Ref)
 	if err != nil {
 		return nil, err
 	}
-	return &Commit{
+	retCommit := &Commit{
 		Files:   commit.Files,
 		Message: commit.GetCommit().GetMessage(),
 		Author:  commit.GetCommit().GetAuthor(),
-	}, nil
+	}
+
+	if commit.Author != nil {
+		if commit.Author.AvatarURL != nil {
```

Comment:
I think what we want is probably to either distinguish between the github author vs git author or we should just assume that we want the github author and remove the `GetCommit` call above and return the author object to use later, removing the need to have the avatar and id fields here.